### PR TITLE
Benchmark for CREATE transactions

### PR DESCRIPTION
**Pull request by libscott**
Benchmark to test throughput of CREATE transactions via the web interface.

Check out the readme: https://github.com/bigchaindb/bigchaindb/pull/1511/files#diff-3c57b0f1abdc1c79ed986908a370ae06

### CODE REVIEW

**Comment by libscott** [comment]
throughput

File: docker-compose.yml [hunk 1]
```diff
@@ -64,11 +64,22 @@ services:
...
     environment:
       BIGCHAINDB_DATABASE_BACKEND: mongodb
       BIGCHAINDB_DATABASE_HOST: mdb
       BIGCHAINDB_DATABASE_PORT: 27017
       BIGCHAINDB_SERVER_BIND: 0.0.0.0:9984
+      BIGCHAINDB_GRAPHITE_HOST: graphite
     ports:
       - "9984"
-    command: bigchaindb start
+    command: bash -c 'bigchaindb -y configure mongodb && bigchaindb start'
```
Comment (sbellem) [inline docker-compose.yml]: `bigchaindb start` should be sufficient since the dockerfile build step takes care of the configuration step

Comment (sbellem) [inline docker-compose.yml]: **Note**: PR #1517 improves the Dockerfile build step so that it is not only mongodb specific, but also supports rethinkdb.

Comment (libscott) [inline docker-compose.yml]: It should be, but for some reason it's not. We can look at it tomorrow if you want.

**Pull request merged**